### ISSUE TITLE
Images of text definition on "significant content" is vague; also link to text definition should not exist!

### ISSUE THREAD

**Issue by mbgower** [w3c/wcag#2693]
A [new ACT](https://www.w3.org/WAI/standards-guidelines/act/rules/0va7u6/proposed/) rule attempts to determine if Images of Text has passed.

However, the [definition of Images of Text](https://www.w3.org/TR/WCAG22/#dfn-images-of-text) in WCAG 2.x contains the following note:
> This does not include [text](https://www.w3.org/TR/WCAG22/#dfn-text) that is part of a picture that contains significant other visual content.

"Significant other visual content" is a very unspecific phrase. As we move to WCAG 3.0, such a phrase does not lend itself to developing a computational assessment.

The ACT rule's tactic is to check if the text is "not the most significant content". This is not very much of an improvement -- and arguably not precisely equivalent to the note in the definition.

> Passed Example 4
[Open in a new tab](https://w3.org/WAI/content-assets/wcag-act-rules/testcases/0va7u6/372b4acc620b897e8fa5e22607121407399f5e07.html)

>This image resource referenced by the object element contains text, but it is not the most significant content.

<object data="/test-assets/0va7u6/times_square.jpg" title="Picture of Times Square, New York"></object>

What would be more useful is to provide computationally testable ways of determining when text that is not central to an image can be ignored.

**Comment by mbgower** [comment]
Suggested wording could include:
> This does not include [text](https://www.w3.org/TR/WCAG22/#dfn-text) that appears in an image but is incidental to the images purpose or function. For example, if the text was hypothetically removed from the image, the image's primary meaning would not be altered, it would be incidental.

**Comment by mbgower** [comment]
In WCAG 3.0, in the context of Images of Text, it could be altered even more to be "This does not include text in an image where the majority of the image content is something other than text."

Most users should be able to rapidly assess that, and an actual calculation of pixels would help decide the edge cases.

**Comment by mbgower** [comment]
Actually, there is another problem in this definition and that is that text in the note links to the definition of text:
> sequence of characters that can be [programmatically determined](https://www.w3.org/TR/WCAG22/#dfn-programmatically-determinable), where the sequence is expressing something in [human language](https://www.w3.org/TR/WCAG22/#dfn-human-language-s)

As it stands right now, any image that contains text would pass this, since none of it is programmatically determinable.

That link to the definition of text should be removed!

**Comment by bruce-usab** [comment]
@mbgower you may be correct about the link to the definition, but I am not sure.

Are you keeping in mind that definition (exception?) also covers incidental text in a photograph.

**Comment by mbgower** [comment]
@bruce-usab The concern on the vague language remains, but not the use of a "text" definition link.

The link in the definition note was removed as part of https://github.com/w3c/wcag/pull/3038
